create_release_pr/action.yml

```
@@ -0,0 +1,70 @@
+name: Automate creating the Release pull request
+description: Automate creating the Release pull request
+inputs:
+  version:
+    description: Current release version
+    required: true
+  token:
+    description: The GitHub PAT for this action to use
+    required: false
+    default: ${{ github.token }}
+  base_branch:
+    description: The base branch for the release pull request, e.g., "main"
+    required: false
+    default: main
+  changelogConfiguration:
```

should we follow the other parameter naming convention too 🤔 

```suggestion
  changelog_configuration:
```

---

Fixed in https://github.com/nimblehq/github-actions-workflows/pull/7/commits/5cd6cc6c63389658f4a68d29b1c7c5169864e128